app/components/Article/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { addHighlight } from 'actions/article';
+import { addHighlight, deleteHighlight, selectHighlight } from 'actions/article';
```

Comment:
also, if we have a highlight module then all the highlight actions should exist in the highlight module.